exampleservice/CustomerService/Events/CustomerRegisteredEvent.cs

```diff
@@ -1,6 +1,7 @@
+using exampleservice.CustomerService.Contract;
 using exampleservice.Framework.BaseFramework;
 
-namespace exampleservice.CustomerService.Contract
+namespace exampleservice.CustomerService.Events
```

Comment:
I used the "Contract" namespace for events because this was the base structure of the "blueprint" (SellTicketService). What was the idea behind the change?

Reply:
the idea was to distinguish more between several different 'things' instead of one big package

Reply:
Sticking to the Blueprint is more important to me that separating it. Also we save some `using` directives which is a hint for better cohesion. I moved it back with the new structure.